### PR TITLE
Added a new directory for items definitions

### PR DESCRIPTION
**Pull request by Hoppelite**
- Added some base stuff on weapons

### CODE REVIEW

File: items/weapons.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# Ranged weapons
+
+|Name           |Description                                                  |Weight |Price  |Dice   |Damage Type  |Traits                         |Range  |
+|---------------|-------------------------------------------------------------|-------|-------|-------|-------------|-------------------------------|-------|
+|Short bow      |A short bow                                                  |2      |25     |2d6    |Piercing     |2-handed                       |120ft  |
+|Crossbow       |A crossbow                                                   |2      |25     |2d6    |Piercing     |2-handed, loading              |120ft  |
+
+# Weapon Traits
+
+## 1-handed
```
Comment (Lambosaurus) [inline items/weapons.md:37]: Something cannot be 2-handed and versatile.
Perhaps the trait should be: 1-handed OR 2-handed OR versatile?

File: items/weapons.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Weapons
```
Comment (Lambosaurus) [inline items/weapons.md:1]: I'd suggest a structural change.
Have:
- systems/items.md - for defining basic items
- systems/weapons.md - for defining basic weapons
And additional tables of items would be outside of the system - ie, setting dependent. These are outside the scope of the system itself.

I am open for discussion on this.

File: items/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Items
+Characters can collect an assortment of many items in their journeys. Here we list some examples, but the extent of what items can exist and how they are used is is for the players and DM to discover.
+
+## General Template
```
Comment (Lambosaurus) [inline items/README.md:4]: Very object oriented.

I agree with the principal, but I think you need to throw away the reference to "general template".
Just define what an item is and has. Weapons should specify that a weapon is an item.

File: items/weapons.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+A list of [weapon traits](#weapon-traits) this weapon has.
+
+### Range
+The maximum effective range on this weapon.
+
+# Melee weapons
+
+|Name           |Description                                                  |Weight |Price  |Dice   |Damage Type  |Traits                         |Range  |
+|---------------|-------------------------------------------------------------|-------|-------|-------|-------------|-------------------------------|-------|
+|Sword          |A sword                                                      |2      |25     |2d6    |Slashing     |1-handed                       |5ft    |
```
Comment (Lambosaurus) [inline items/weapons.md:23]: In the attack systems I define an attack as two times the weapon dice. The weapon dice is simply d6 or d8.
If this is unclear, then perhaps the weapons system should be ammended.

File: items/weapons.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+This weapon can be wielded with only one hand
+
+## 2-handed
+This weapon requires two hands to wield
+
+## Versatile
+This weapon can be wielded with a second hand to increase its dice
+
+## Throwing
+This weapon can be thrown without [disadvantage](../systems/rolls.md#disadvantage). When thrown, [Range](#range) is 20ft.
```
Comment (Lambosaurus) [inline items/weapons.md:47]: Can a great axe be thrown for say.. 2d10 -1d4 damage? I wonder if a two handed weapon should be double disadvantage when thrown.
This may be too complicated.

**Review by Lambosaurus — CHANGES_REQUESTED** [review]
Some minor tweaks, and proposing to remove the directory.

File: items/weapons.md (hunk 4 above)
Comment (Hoppelite) [reply items/weapons.md:23]: I thought it was like that, but also thought this might add clarity.

File: items/weapons.md (hunk 5 above)
Comment (Hoppelite) [reply items/weapons.md:47]: I guess if it is out of range (5ft) does it always fail? Or maybe we stack more disadvantages on it.

File: items/weapons.md (hunk 2 above)
Comment (Hoppelite) [reply items/weapons.md:1]: Hmm, yea. I was just trying to not have all of everything in one file. I was thinking there would be a file for each item type which would make things messy, but we could do it like this, and maybe put general items in `items.md`

File: items/weapons.md (hunk 1 above)
Comment (Hoppelite) [reply items/weapons.md:37]: Maybe the handedness shouldn't be a trait, and is instead a fixed setting, seeing how all items will define it. I am not 100% sold on the versatile stuff yet either. Like do we just allow any 1-handed weapon to be wielded in 2 for extra effect?

Comment (Lambosaurus) [reply items/weapons.md]: Opinion a)
I dont know. I think it takes away from those weapons being special?
Opinion b)
Make it a feat. If you have bludgeoning master, all 1 handed bludgeoning weapons are versatile to you.
Opinion c) Idk, ask ollie.

File: items/weapons.md (hunk 5 above)
Comment (Lambosaurus) [reply items/weapons.md]: Well, I do really like being able to throw other weapons.

File: items/weapons.md (hunk 2 above)
Comment (Lambosaurus) [reply items/weapons.md]: Yes. If we have additional lists of items, such as lists of potions, ect, then we can move it to items/potions.md.
But imo the base items.md should be in systems/.

**Review by Lambosaurus — APPROVED** [review]
